release/build_release.sh

```diff
@@ -2,6 +2,7 @@
 
 source _bom.sh
 source _git.sh
+source _github.sh
```

Comment:
@kiwm, why I need this here since I'm not calling any function declared on the file `_github.sh`? Can you please clarify?